draft-cenzano-moq-media-interop.md

```diff
@@ -63,6 +63,8 @@ to other formats such as FMP4.
 The publisher selects a namespace of their choosing, and sends an ANNOUNCE
 message for this namespace.
 
+Within the publisher namespace the publisher can send any number of audio or/and video tracks with the names of their choosing. The subscriber will consider all of those as part of the same synchronization gproup (timestamps aligned to the same timeline).
+
```

Comment:
nit: 80 char line limit for `make`

Bigger question: how does the subscriber know what to subscribe to if the track names are not fixed?

Reply:
I thoght we had catalog for those type of "out of band" communication. This will be always true for any type of MOQT connections.
Perhaps we can recommend some naming convention in case there is NO catalog invoved, such as : `video0000`, `audio0000`, `video0001`, `audio0001`,...

Reply:
Let's assume this is used without catalogs.  I like using a sequence number (maybe just 0, 1, 2 ... so there's no limit).  So there will be at least one audio and one video called `audio0` and `video`, additional tracks may be available (you have to subscribe to find out)

Reply:
Yes, in this use case (media and NO catalog) sequence number should be sequentiantial, if you get error "0x3 | Track Does Not Exist" you know there is NO more.

It is a bit weak, but without catalog I understand the subsciber already knows (more or less) what to expect